internal/cli/atlas/clusters/create.go

```diff
@@ -225,7 +223,7 @@ Some of the cluster configuration options are available via flags, but for full
 		},
 		Annotations: map[string]string{
 			"args":            "clusterName",
-			"clusterNameDesc": "Name of the cluster. The cluster name cannot be changed after the cluster is created. Cluster name can contain ASCII letters, numbers, and hyphen.",
+			"clusterNameDesc": "Name of the cluster. The cluster name cannot be changed after the cluster is created. Cluster name can contain ASCII letters, numbers, and hyphens.",
```

Comment:
not for this PR but should we add an annotation to link tot he API docs? this is something we kind of lost with generated docs and for this particular case it may be easier to link to the api docs to see the file format

Reply:
I agree, I think that'd be a good idea.